src/amo/components/SearchForm/index.js

```diff
@@ -144,7 +144,9 @@ export class SearchFormBase extends React.Component {
         filters,
       }));
     });
-  }, 200)
+  // Configure debounce (200ms) with `leading: true` so that the method is
+  // triggered immediately and subsequent calls are debounced.
+  }, 200, { leading: true, trailing: false })
```

Comment:
Is this a value we would want to use elsewhere in the app? It seems like 200ms is a good default for debouncing. Maybe we want to store this as a constant somewhere?

Reply:
The `AddonReview` component (the only other one that uses `debounce` so far) is configured with 800ms.

Reply:
Ah, fair enough. Okay then, not worth it 😄